source/tests/unit/ni_fake_service_tests.cpp

```diff
@@ -148,6 +175,39 @@ TEST(NiFakeServiceTests, NiFakeService_InitWithOptionsThenClose_SessionIsClosed)
   EXPECT_EQ(0, session_repository.access_session(session.id(), ""));
 }
 
+TEST(NiFakeServiceTests, NiFakeService_InitExtCalThenCloseExtCal_SessionIsClosed)
```

Comment:
Note: This test is similar to NiFakeService_InitWithOptionsThenClose_SessionIsClosed found above.